File: dev/20190402/k2works/sam-app/FizzBuzzFunction/src/main/java/fizzbuzz/domain/type/FizzBuzzType02.java
package fizzbuzz.domain.type;

import fizzbuzz.domain.model.FizzBuzzValue;

public class FizzBuzzType02 extends IFizzBuzzType {
    @Override
    public FizzBuzzValue generate(int number) {
        return new FizzBuzzValue(number,Integer.toString(number));
    }
}

File: dev/20190822/k2works/src/Domain/Type/FizzBuzzType02.ts
import { FizzBuzzValue } from "../Model/FizzBuzzValue";
import { IFizzBuzzType } from "./IFizzBuzzType";

export class FizzBuzzType02 extends IFizzBuzzType {
  public generate(input: number): FizzBuzzValue {
    return new FizzBuzzValue(input, input.toString());
  }
}
